package.json

```diff
@@ -116,6 +116,7 @@
     "karma-jasmine-html-reporter": "~2.0.0",
     "karma-read-json": "1.1.0",
     "karma-typescript": "5.5.3",
+    "moment": "^2.29.4",
```

Comment:
`moment` was extraneously imported from `@energyweb/onchain-claims`, but now onchain-claims does not install `moment`
Also `moment` now in maintanence mode. Need to replace it

Reply:
onchain-claims does not need it. I added it because it is used in switchboard https://github.com/energywebfoundation/switchboard-dapp/blob/63f96fd4c1a42c47750a815cb1456b2dc9d81896/angular.json#L70
I thought that it is dev dependency, because it is used only to build

Reply:
yes, build failing without moment. I will try to remove it

Reply:
But this is not connected at all. This info is based on data from metamask, not moment js. 
I tested it locally on your branch by removing script and moment js and everything works fine.
Are you sure you're connected to proper chain?

Reply:
it works without moment. problem was with cookies